packages/graphql/tests/integration/issues/1756.int.test.ts

```diff
@@ -165,4 +165,4 @@ describe("https://github.com/neo4j/graphql/issues/1756", () => {
             ],
         });
     });
-});
+});
```

Comment:
**non-blocking** Missing newline at the end of file, not important, but make sure you have prettier configured in your setup, as this should not happen as far as I'm aware

Reply:
I have it set to format on typing but for some reason it doesn't seem to alway catch the newline at the ends of file